### ISSUE TITLE
[QUESTION] [1.7.10] Forestry pack update to Forestry 4

### ISSUE THREAD

**Issue by hron84** [jaquadro/StorageDrawers#397]
I know right 1.7.10 is not a thing anymore, but some of us stuck on this version. Is there any hope for update your extra drawers pack to Forestry 4? If not, could you please help me with some pointers how can I work on it? I'm not have too much free time, but as far as I see there is only a recipe problem with it, so it shouldn't be too hard to be fixed (drawers render correctly in NEI, so there is no texture/rendering problem, just they have no recipe). 

**Update:** there is some drawers that have a recipe some don't. E.g. Plum, Papaya, Poplar, Pine drawers has no recipe, but Teak, Willow, Wenge, Lime has a recipe but this is just a wild poking around.

**Comment by bookerthegeek** [comment]
An easy bandaid would be minetweaker recipes.

**Comment by hron84** [comment]
@bookerthegeek yeah, i thinking about that, but that means a bunch of recipes and a quite lot of poking as Forestry brings too much wood.

**Comment by bookerthegeek** [comment]
Not really, I think I already did it for my private pack.  I'll look and see.
